schemas/EGA.individual.json

```diff
@@ -41,20 +41,47 @@
       "minimal_public_attributes":{
         "type": "object",
         "title": "Minimal public attributes describing an individual",
-        "description": "Among all attributes describing an individual, some may contain identifiable metadata and thus must be private. Nevertheless, there are three required attributes (even if they are unknown): subject id, biological sex and phenotype. These shall be displayed and queryable.",
+        "description": "Among all attributes describing an individual, some may contain identifiable metadata and thus must be private. Nevertheless, there are three/four required attributes (even if they are unknown): subject id, biological sex and phenotype. These shall be displayed and queryable. In the case of a healthy individual (with no phenotypic abnormalities nor diseases), the 'phenotypes' and 'diseases' arrays will contain a reference to 'Unaffected' [NCIT:C94232].",
         "additionalProperties": false,
-        "required": ["subject_id", "biological_sex", "phenotype"],
+        "required": ["subject_id", "biological_sex"],
```

Comment:
Just double checking :)  In the description in the ticket, it is mentioned that "subject_id, phenotype and sex" are mandatory fields, but phenotype is not included in required.

Reply:
I found them below :)